desktop-ui/presentation/presentation.cpp

```diff
@@ -417,7 +417,13 @@ auto Presentation::loadEmulator() -> void {
 
     Group peripheralGroup;
     { MenuRadioItem peripheralItem{&portMenu};
+      peripheralItem.setAttribute<ares::Node::Port>("port", port);
       peripheralItem.setText("Nothing");
+      peripheralItem.onActivate([=] {
+        auto port = peripheralItem.attribute<ares::Node::Port>("port");
+        port->disconnect();
+        port->allocate(); // deallocation for ports not implementing disconnect()
```

Comment:
Is this required? The default disconnect implementation removes the connected peripheral node see https://github.com/ares-emulator/ares/blob/3ca1f9ebb4ae3f472f7fba661746058f84126536/ares/ares/node/port.hpp#L32